docs/community/release-notes.md

```diff
@@ -34,6 +34,89 @@ You can determine your currently installed version using `pip show`:
 
 ---
 
+## 3.15.x series
+
+### 3.15.0
+
+Date: ...
+
+* Added jQuery 3.7.1 support & dropped previous version [[#9094](https://github.com/encode/django-rest-framework/pull/9094)]
```

Comment:
```suggestion
* Django 5.0 and Python 3.12 support [[#9157] (https://github.com/encode/django-rest-framework/pull/9157)]
* Use POST method instead of GET to perform logout in browsable API [[9208] (https://github.com/encode/django-rest-framework/pull/9208)]
* Added jQuery 3.7.1 support & dropped previous version [[#9094](https://github.com/encode/django-rest-framework/pull/9094)]
```